README.md

```diff
@@ -1,6 +1,8 @@
 # DistributionsAD.jl
 
-[![ForwardDiff and Tracker tests](https://github.com/TuringLang/DistributionsAD.jl/workflows/ForwardDiff%20and%20Tracker%20tests/badge.svg?branch=master)](https://github.com/TuringLang/DistributionsAD.jl/actions?query=workflow%3A%22ForwardDiff+and+Tracker+tests%22)
+![Zygote tests](https://github.com/TuringLang/DistributionsAD.jl/workflows/Zygote%20tests/badge.svg)
+
+![ForwardDiff tests](https://github.com/TuringLang/DistributionsAD.jl/workflows/ForwardDiff%20tests/badge.svg)
```

Comment:
One should add a badge for the separate Tracker tests as well, and only display the most recent CI run on the master branch (as explained above).